### PR TITLE
Opening in progress applications 

### PR DESCRIPTION
**Pull request by Harriethw**
We would like users to be able to open the applications they have started.
This PR:
* renames the `new` applications page to be the `index` (there will be a new `new` page created, where the CRN form will move to)
* adds a link to the application on the applicant's name.

### CODE REVIEW

File: integration_tests/pages/apply/list.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  }
+
+  private shouldShowApplications(applications: Array<Cas2ApplicationSummary>): void {
+    applications.forEach(application => {
+      cy.contains(application.person.name)
+        .should('have.attr', 'href', paths.applications.show({ id: application.id }))
+        .parent()
+        .parent()
+        .within(() => {
+          cy.get('th').eq(0).contains(application.person.name)
```
Comment (patrickjfl) [inline integration_tests/pages/apply/list.ts:31]: Do we gain anything from this check? Seems to me like we've already checked this on line 26...

File: server/utils/taskListUtils.test.ts [hunk 2]
```diff
@@ -67,5 +67,13 @@ describe('taskListUtils', () => {
         '<strong class="govuk-tag govuk-tag--grey app-task-list__tag" id="second-task-status">Cannot start yet</strong>',
       )
     })
+
+    it('returns a completed tag when the task cannot be started', () => {
```
Comment (patrickjfl) [inline server/utils/taskListUtils.test.ts:71]: I think the test name is wrong here (and also on line 47, by the looks)

Comment (patrickjfl) [reply server/utils/taskListUtils.test.ts:71]: or in fact, it looks like the test on line 47 should be deleted

**Review by patrickjfl — APPROVED** [review]
Looks good to me! Just a couple of minor comments 👍

File: integration_tests/pages/apply/list.ts (hunk 1 above)
Comment (Harriethw) [reply integration_tests/pages/apply/list.ts:31]: ah really good spot. I've now copied CAS3's way of doing this which I think is a bit better and more precise:
``` 
cy.get(`a[href*="${paths.applications.show({ id: application.id })}"]`)
        .parent()
        .parent()
        .within(() => {
          cy.get('th').eq(0).contains(application.person.name)
        })
 ```

Comment (patrickjfl) [reply integration_tests/pages/apply/list.ts:31]: looks good!

**Pull request merged**